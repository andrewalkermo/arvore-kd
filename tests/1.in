7
jose um
livro de jose
1950
livrodejose.txt
maria dois
livro de maria
1974
livrodemaria.txt
joana tres
livro de joana
1962
livrodejoana.txt
alberto quatro
livro de alberto
1955
livrodealberto.txt
manoel cinco
livro de manoel
1998
livrodemanoel.txt
daniel seis
livro de daniel
1948
livrodedaniel.txt
karen sete
livro de karen
1935
livrodekaren.txt
i
jose segundo
livro de jose s.
1949
livrodejoses.txt
i
jose terceiro
livro de jose t.
1949
livrodejoset.txt
p
2
c
jose um
c
jose quarto
n
josa
josi
n
manuel
ze
n
tom
ze
a
1962
1971
a
2000
2022
q
manuel
ze
1930
1974
t
e
