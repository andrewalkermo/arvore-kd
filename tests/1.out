arvore k-d gerada
inserido registro com nome: jose segundo
inserido registro com nome: jose terceiro
pagina: 2
jose um
livro de jose
1950
livrodejose.txt
joana tres
livro de joana
1962
livrodejoana.txt
jose segundo
livro de jose s.
1949
livrodejoses.txt
jose terceiro
livro de jose t.
1949
livrodejoset.txt
nome: jose um
livro de jose
1950
livrodejose.txt
nao foi encontrado registro com nome: jose quarto
jose um
livro de jose
1950
livrodejose.txt
jose segundo
livro de jose s.
1949
livrodejoses.txt
jose terceiro
livro de jose t.
1949
livrodejoset.txt
maria dois
livro de maria
1974
livrodemaria.txt
nao foi encontrado registro com nome entre: "tom" e "ze"
joana tres
livro de joana
1962
livrodejoana.txt
nao foi encontrado registro com ano entre: 2000 e 2022
maria dois
livro de maria
1974
livrodemaria.txt
nome: alberto quatro fesq: pagina fdir: 1948
ano: 1948 fesq: pagina fdir: pagina
ano: 1962 fesq: alberto quatro fdir: pagina
nome: jose um fesq: 1962 fdir: 1974
nome: karen sete fesq: pagina fdir: pagina
ano: 1974 fesq: karen sete fdir: manoel cinco
nome: manoel cinco fesq: pagina fdir: pagina
